explorer/src/pages/SecurityDetailsPage.tsx

```diff
@@ -0,0 +1,324 @@
+    type: DisplayType.URL,
+  },
+  {
+    display: "Preferred Languages",
+    key: "preferred_languages",
+    type: DisplayType.String,
+  },
+  {
+    display: "Source Code URL",
+    key: "source_code",
+    type: DisplayType.URL,
+  },
+  {
+    display: "Encryption",
+    key: "encryption",
+    type: DisplayType.PGP,
+  },
+  {
+    display: "Auditors",
+    key: "auditors",
+    type: DisplayType.Auditors,
+  },
+  {
+    display: "Acknowledgements",
+    key: "acknowledgements",
+    type: DisplayType.URL,
+  },
+  {
+    display: "Expiry",
+    key: "expiry",
+    type: DisplayType.Date,
+  },
+];
+
+function SecurityDetails({
+  pubkey,
+  info,
+}: {
+  pubkey: PublicKey;
+  info?: CacheEntry<Account>;
+}) {
+  const fetchAccount = useFetchAccountInfo();
+
+  if (!info || info.status === FetchStatus.Fetching) {
+    return <LoadingCard />;
+  } else if (
+    info.status === FetchStatus.FetchFailed ||
+    info.data?.lamports === undefined
+  ) {
+    return <ErrorCard retry={() => fetchAccount(pubkey)} text="Fetch Failed" />;
+  }
+  const account = info.data;
+
+  const data = account?.details?.data;
+  if (!data || data.program !== "bpf-upgradeable-loader" || !data.programData) {
+    return <ErrorCard text="Account is not a program" />;
+  }
+
+  const securityTXT = fromProgramData(data.programData);
+
+  if (!securityTXT) {
+    return <ErrorCard text="Account has no security.txt" />;
+  }
+
+  return (
+    <div className="card security-txt">
+      <div className="card-header">
+        <h3 className="card-header-title mb-0 d-flex align-items-center">
+          Overview
+        </h3>
+      </div>
+      <TableCardBody>
+        {ROWS.filter((x) => x.key in securityTXT).map((x, idx) => {
+          return (
+            <tr key={idx}>
+              <td className="w-100">{x.display}</td>
+              <RenderEntry value={securityTXT[x.key]} type={x.type} />
+            </tr>
+          );
+        })}
+      </TableCardBody>
+    </div>
+  );
+}
+
+function RenderEntry({
+  value,
+  type,
+}: {
+  value: SecurityTXT[keyof SecurityTXT];
+  type: DisplayType;
+}) {
+  if (!value) {
+    return <></>;
+  }
+  switch (type) {
+    case DisplayType.String:
+      return <td className="text-lg-end font-monospace">{value}</td>;
+    case DisplayType.Contacts:
+      return (
+        <td className="text-lg-end font-monospace">
+          <ul>
+            {value?.split(",").map((c, i) => {
+              const idx = c.indexOf(":");
+              const [type, information] = [c.slice(0, idx), c.slice(idx + 1)];
+              return (
+                <li key={i}>
+                  <Contact type={type} information={information} />
+                </li>
+              );
+            })}
+          </ul>
+        </td>
+      );
+    case DisplayType.URL:
+      if (isValidLink(value)) {
+        return (
+          <td className="text-lg-end">
+            <span className="font-monospace">
+              <a href={value}>
```

Comment:
Please add `rel="noopener noreferrer" target="_blank"`

Reply:
Should be fixed now